matrix_sdk/Cargo.toml

```diff
@@ -31,7 +31,6 @@ matrix-sdk-common = { version = "0.1.0", path = "../matrix_sdk_common" }
 [dependencies.matrix-sdk-base]
 version = "0.1.0"
 path = "../matrix_sdk_base"
-default_features = false
```

Comment:
I think I meant add the `messages` feature to the default features in the matrix-sdk. This enables all features and we can't disable them using our feature flags, no?